content/en/docs/ambient/usage/waypoint/index.md

```diff
@@ -9,4 +9,22 @@ owner: istio/wg-networking-maintainers
 test: no
 ---
 
-This page is under construction.
+Ambient splits Istio’s functionality into two distinct layers, a secure overlay layer and a Layer 7 processing layer.
+The waypoint proxy is an optional component that is Envoy-based and handles L7 processing for different resources.
+What is unique about the waypoint proxy is that it runs outside of the application pod. A waypoint proxy can install,
+upgrade, and scale independently from the application, as well as reduce operational costs. When deploying a waypoint,
+you can configure the waypoint to process traffic for different resource types such as `service` or `workload` or `all`.
```

Comment:
`none` too (for testing purposes)

Reply:
And maybe mention that `service` is default.

Reply:
mentioned multiple times below :) didn't feel we need to mention none here as it is mainly for testing but it was mentioned in the `Deploy a waypoint proxy` section